packages/fxa-settings/src/pages/Index/index.tsx

```diff
@@ -55,14 +160,49 @@ export const Index = ({
           {...{ clientId, serviceName }}
         />
       )}
-      <FtlMsg id="index-email-input" attrs={{ label: true }}>
-        <InputText className="mt-8" type="email" label="Enter your email" />
-      </FtlMsg>
-      <div className="flex mt-5">
-        <button className="cta-primary cta-xl" type="submit">
-          <FtlMsg id="index-cta">Sign up or sign in</FtlMsg>
-        </button>
-      </div>
+      {errorBannerMessage && (
+        <Banner
+          type="error"
+          content={{ localizedHeading: errorBannerMessage }}
+        />
+      )}
+      {successBannerMessage && (
+        <Banner
+          type="success"
+          content={{ localizedHeading: successBannerMessage }}
+        />
+      )}
+
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <FtlMsg id="index-email-input" attrs={{ label: true }}>
+          <InputText
+            className="mt-8"
+            name="email"
+            label="Enter your email"
+            inputRef={register({ required: true })}
+            // value={prefillEmail}
```

Comment:
Shouldn't need placeholder or value here (in fact, value should likely be removed from InputText prop types altogether), but setting the default value this way doesn't quite work (the prefill email isn't visible in the input). Instead it might be better to use react-hook-form's useForm with defaultValues